doc/api/cli.md

```diff
@@ -624,6 +624,18 @@ Preload the specified module at startup.
 Follows `require()`'s module resolution
 rules. `module` may be either a path to a file, or a node module name.
 
+### `--unhandled-rejections=mode`
+<!-- YAML
+added: REPLACEME
+-->
+
+Using this flag allows to specify what should happen when an unhandled rejection
+occurs. One of three modes can be chosen:
+
+* `strict`: Raise the unhandled rejection as uncaught exception.
+* `warn`: The same as the default but without the deprecation warning.
+* `none`: Silence all warnings.
```

Comment:
Can you indicate which the default mode is (`warn`)?

Reply:
The documentation is correct as is currently. Non is the real default and I outline in the `warn` description that it is the same as the default with a tiny difference.

Reply:
IIUC, there is no option for the default mode.

Reply:
reading this as it is now, i would expect node to be completely silent if i don't pass any flags.

Reply:
@devsnek I am surprised by that. `warn` clearly indicated that it is the default but without a deprecation warning and `none` indicates that it would `silence` warnings which implies that there are indeed warnings by default.

Reply:
I am happy for suggestions to reword.

Reply:
@BridgeAR sorry i should clarify, reading this tells me that warn is the same as "the default" (which is not defined in this context) but without some warning.

i'd suggest removing the fourth mode because confusion is confusing and then putting
```
* `warn` (*default*) Log a warning containing the unhandled rejection value
```

Reply:
One of the goals of this PR was to keep everything existing working exactly as it was before, if no flag was used.

So making `warn` the default implies therefore that the deprecation message will also be printed when using the flag, which I do not think is a good idea.

Reply:
I agree leaving behavior is make sense.  I do agree that the doc could be clarified so that it's more obvious what the behavior is if you don't specify the option.

Maybe by adding a line above "Using this flag.." which says something like. "By default will do xxx" and then changing the next line to be "Using this flag changes what happens when an unhandled rejection occurs"

Reply:
I added extra documentation as suggested and I believe now it's clearer that it's not identical to the default.